test/fixtures/compile-test-data.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
When you say "it is" do you mean it is run by another command somewhere and I just didn't find it?